themes/bootstrap3/templates/RecordTab/overdriveholdings.phtml

```diff
@@ -0,0 +1,182 @@
+<?php $access = $this->driver->getOverdriveAccess();?>
+<?php if ($access->code == 'od_account_noaccess'): ?>
+  <div class="hold">
+    <div class="alert alert-warning">
+      <?= $this->translate('od_code_contentnotavail') ?>
+    </div>
+  </div>
+<?php elseif ($access->code == 'od_code_login_for_avail'): ?>
+  <p class='alert-info'><strong>
+      <?= $this->transEsc('od_code_login_for_avail') ?>
+    </strong></p>
+<?php else:
+  $od_id = $this->driver->getOverdriveID();
+  $rec_id = $this->driver->getUniqueID();
+  $hold_url = $this->url('overdrive-hold');
+  $avail = $this->driver->getOverdriveAvailability();
+  $checkedOutResult = $this->driver->isCheckedOut();
+  $checkedOut = $checkedOutResult->data;
+  if (!$checkedOut):
+    $hold = $this->driver->isHeld();
+  endif; ?>
+
+  <div class="internetDetails">
+    <div class="odbrand pull-left media-left">
+      <img
+        src=""<?=$this->imageLink('OverDrive_Logo_28x28_rgb.jpg')?>"">
+    </div>
+    <h3>
+      <?= $this->transEsc('Online Access') ?>
+    </h3>
+    <div class="availability">
+      <p>
+        <?php if ($avail->data->copiesAvailable > 0): ?>
+          <span class="text-success">
+            <?= $this->icon('status-available', 'icon-link__icon') ?>
+          </span>
+        <?php else: ?>
+          <span class="text-danger">
+            <?= $this->icon('status-unavailable', 'icon-link__icon') ?>
+          </span>
+        <?php endif; ?>
+
+        <?php if (isset($avail->data->availabilityType) && $avail->data->availabilityType == 'AlwaysAvailable'): ?>
+          <?= $this->transEsc('od_unlimited') ?>
+        <?php else: ?>
+          <?= $avail->data->copiesAvailable . '/' . $avail->data->copiesOwned ?>
+        <?php endif; ?>
+        <?= $this->transEsc('od_copies_available', ['%%copies%%' => $copies]) ?>
+        <?php if ($avail->data->numberOfHolds > 0): ?>
+          <?= '; ' . $this->transEsc('od_waiting', ['holds' => $avail->data->numberOfHolds], null, true) ?>
+        <?php endif; ?>
+      </p>
+
+      <?php if (!$checkedOut || $checkedOutResult->isMagazine): ?>
+        <?php if (isset($avail->data->copiesAvailable) && $avail->data->copiesAvailable > 0): ?>
+          <p>
+            <a class="btn btn-primary checkout" data-lightbox title="<?= $this->transEsc('request_place_text') ?>"
+              href="<?= "$hold_url?od_id=$od_id&rec_id=$rec_id&action=checkoutConfirm" ?>">
```

Comment:
I think I've asked this question elsewhere, but this is another place where I wonder if we need to urlencode some of these IDs just in case they contain unexpected characters. But maybe it's not necessary if they're absolutely guaranteed to be integers.